### ISSUE TITLE
Configuration doesn't save

### ISSUE THREAD

**Issue by KB8RCO** [Xastir/Xastir-Qt#5]
I understand this is still preliminary, but the basic configuration (call, SSID, passcode, location) does not seem to save.  It does keep the call and configuration for the interface. 

Are updates planned, or has this been dropped?

**Comment by godfreja** [comment]
I'm not aware of anyone actively working on it.

**Comment by tvrusso** [comment]
I don't believe any work has been done on this in almost a year, and then it was only to pull it out of the main Xastir project and into its own repo.  As it barely does more than monitor packets, it's not that useful, and there hasn't been anyone tripping over themselves to add functionality.

Something is clearly amiss in the stationsettings.cpp functions that save configuration --- there are saved bits of configuration such as lat/lon and station symbol but they appear to be getting written out as `@Variant` data and are either not being written correctly or not being read back in correctly.
I find lat/lon being incorrect, and the station symbol being only half right (I have \X set and get only the X back on restart).  The station with ssid is right, but nothing else.  (the settings, by the way, are saved in "~/.config/Xastir Group/Xastir QT.conf" and you can see for yourself what's being incorrectly saved).

I find that all the interface configuration is correctly saved, though, including passcode and filter.  

My config file winds up looking like:

```
[DeviceInterfaces]
Interface0\activateOnStartup=true
Interface0\allowTransmit=false
Interface0\callsign=KM5VY
Interface0\filter=r/35/-106/300\n
Interface0\hostName=bogopi
Interface0\interfaceClass=NetInterface
Interface0\passcode=21753
Interface0\port=14580

[StationSettings]
callsign=KM5VY-14
comment=
group=@Variant(\0\0\0\a\0\0)
hasPHG=false
lat=@Variant(\0\0\0\x87\0\0\0\0)
lon=@Variant(\0\0\0\x87\0\0\0\0)
symbol=@Variant(\0\0\0\a\0X)
```

**Comment by tvrusso** [comment]
Oh, yeah.  stationconfiguration doesn't even *look* at the lat/lon setting in the dialog.  It never actually gets set.  There are no hooks in the UI to update anything if you enter data in the dialog for lat/lon.  There's a comment there that makes it look like it just got left out to be figured out later.

So it makes sense that nothing gets saved --- the lat/lon part of the station settings is never set anywhere.  This is not likely to get changed much unless someone is inspired to start working on Xastir-Qt again.
